File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        Set<Integer> st = new HashSet<>();
        
        for(int i : nums){
            st.add(i);
        }
        
        List<Integer> lst = new ArrayList<>();
        for(int i = 1; i <= nums.length; i++){
            if(!st.contains(i))
                lst.add(i);
        }
        
        return lst;
    }
}

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        s = set()
        ans = set()
        
        nums.sort()
        
        for i in range(len(nums)):
            s.add(nums[i])
            
        for i in range(1,len(nums)+1):
            if i not in s:
                ans.add(i)
                
        return ans;
            
        
        